modules/account-lib/src/coin/stx/transactionBuilder.ts

```diff
@@ -0,0 +1,251 @@
+import BigNumber from 'bignumber.js';
+import BigNum from 'bn.js';
```

Comment:
Why do we need two bignumber libs?

Reply:
Stacks lib uses `BigNum` from `bn.js` while signature of `abstract validateValue(value: BigNumber): void;` in BaseTransactionBuilder has `BigNumber`